spec/services/calculation_orchestrator_spec.rb

```diff
@@ -50,15 +50,15 @@
         },
       ],
       uplift: {
-        sub_total: 300.0,
         per_participant: 100.0,
+        sub_total: 1000.0,
       },
     }
   end
 
   before do
     10.times do
-      participant_declaration = create(:participant_declaration, lead_provider: call_off_contract.lead_provider)
+      participant_declaration = create(:participant_declaration, :sparsity_uplift, lead_provider: call_off_contract.lead_provider)
```

Comment:
Also would test with pupil_premium_uplift and a combination of the two.

Reply:
Might be okay as-is if we expand the unit test (the above turnip feature) to test both.

Reply:
I'd do it in the spec explicitly. Turnip scenarios are more suitable for end-to-end integration tests as and such are much more fragile than testing just the participant_declaration.